insight-engine/latest/install/options.md

```diff
@@ -203,9 +203,9 @@ This task assumes you have:
 
     ```text
     index.subsystem.name=solr6
-    solr.secureComms=secret
-    solr.port=8983
-    solr.sharedSecret=password
+    mutual-TLS-solr.secureComms=https
```

Comment:
I can't find any information about this property name being changed. Where are you getting this from?

Reply:
@tpage-alfresco  it looks like it has come from this PR https://github.com/Alfresco/docs-alfresco/pull/312
If you dont agree with the PR I am happy to close it !

Reply:
I don't understand your reply. That PR doesn't mention a property called `mutual-TLS-solr.secureComms`.

Reply:
I have made this the same as the others.